File: tests/fixtures/code_samples/py/sample_060.py
"""Sample 60: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_60(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 60."""
    total = 60
    for x in xs:
        total += int(x)
    return total


def operation_60_pure(value: int) -> int:
    return (value * 60) % 7919

File: tests/fixtures/code_samples/ts/sample_060.ts
/** Sample 60: small utility. */

export function operation60(xs: number[]): number {
  let total = 60;
  for (const x of xs) total += x;
  return total;
}

export function operationPure60(value: number): number {
  return (value * 60) % 7919;
}

